lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -79,11 +79,7 @@ describe('ContentSelector', () => {
     const noop = () => {};
     return mount(
       <Config.Provider value={fakeConfig}>
-        <ContentSelector
-          setErrorInfo={noop}
-          onSelectContent={noop}
-          {...props}
-        />
+        <ContentSelector onError={noop} onSelectContent={noop} {...props} />
```

Comment:
I found that the the `setErrorInfo` prop was incorrect. Currently, the prop is called `onError`.